cmd/cluster-agent/app/config.go

```diff
@@ -77,5 +77,5 @@ func initRuntimeSettings() error {
 		return err
 	}
 
-	return commonsettings.RegisterRuntimeSetting(commonsettings.ProfilingRuntimeSetting("internal_profiling"))
+	return commonsettings.RegisterRuntimeSetting(commonsettings.ProfilingRuntimeSetting{SettingName: "internal_profiling", Service: "datadog-cluster-agent"})
```

Comment:
Previously the cluster agent would set a service name based on the agent flavor [here](https://github.com/DataDog/datadog-agent/blob/41a54d0772ee937f533f47207a10b13b27a7658d/pkg/config/settings/runtime_setting_profiling.go#L75), so this should preserve existing behavior